study-builder/studies/rarex/emails/data-sharing-complete-reminder-proxy.html

```diff
@@ -0,0 +1,160 @@
+<html>
```

Comment:
All the new `*-proxy.html` files are basically copy-paste of existing files, where the only change is the salutation `Dear -ddp.proxy.firstName- -ddp.proxy.lastName-,`.